### DISCUSSION TITLE
关于模板管理的应用

### DISCUSSION THREAD

**Discussion by KaiN019-fool** [WatsonYao/nian_app#106] [Q&A] [answered]
我发现nian里设计了模板管理的功能，但是并没有在记本中发现可以调用该功能的入口，有点疑惑它的使用方式，辛苦解答₍˄·͈༝·͈˄*₎◞ ̑̑

**Comment by llwy142086** [comment, marked as answer]
根據我自己用過的功能來看，可以在標準計本和點評集中選擇，其他還有哪些地方能用我也不是很清楚。
![Screenshot_2024-11-11-02-35-10-244_sa nian so-edit](https://github.com/user-attachments/assets/d9b4e158-a647-4845-9ee4-e7b3c719698d)
![Screenshot_2024-11-11-02-35-35-291_sa nian so-edit](https://github.com/user-attachments/assets/4102e4b2-4b00-4be2-9fef-d41c2f872e21)

**Reply by WatsonYao** [reply]
是的，目前可以调用的地方如上面图片所示。需要补充说明的是，写进展的时候，右上角按钮区域是可以左右滑动的；如果是早期用户升级上来的版本，需要打开一次「实验室」-「进展编辑页-自定义工具栏」。

**Reply by KaiN019-fool** [reply]
原来如此，我之前没有注意到上面的工具栏是可以滑动的，谢谢大家🥹🥹好伟大的设计！！
